x/dex/client/cli/tx_place_limit_order.go

```diff
@@ -91,7 +106,8 @@ func CmdPlaceLimitOrder() *cobra.Command {
 				amountInInt,
 				orderType,
 				goodTil,
-				maxAmountOutIntP,
+				&maxAmountOutInt,
+				&priceDec,
```

Comment:
since you declare variables as `var priceDec math_utils.PrecDec` and `var maxAmountOutInt math.Int` the arguments never nil, does not it brake some logic inside PlaceOrder handler?